dali/python/nvidia/dali/pipeline.py

```diff
@@ -172,13 +174,11 @@ def create_pipeline():
     number of outputs from the pipeline.
 """
 
-    def __init__(self, batch_size=-1, num_threads=-1, device_id=-1, seed=-1,
-                 exec_pipelined=True, prefetch_queue_depth=2,
-                 exec_async=True, bytes_per_sample=0,
-                 set_affinity=False, max_streams=-1, default_cuda_stream_priority=0,
-                 *,
-                 enable_memory_stats=False, py_num_workers=1, py_start_method="fork",
-                 py_callback_pickler=None, output_dtype=None, output_ndim=None):
+    def __init__(self, batch_size=-1, num_threads=-1, device_id=-1, seed=-1, exec_pipelined=True,
+                 prefetch_queue_depth=2, exec_async=True, bytes_per_sample=0, set_affinity=False,
+                 max_streams=-1, default_cuda_stream_priority=0, *, enable_memory_stats=False,
+                 py_num_workers=1, py_start_method="fork", py_callback_pickler=None,
+                 output_dtype=None, output_ndim=None):
```

Comment:
Logical separation is defeated. I think the original formatting was OK...

Reply:
Done with one arg per line.